### DISCUSSION TITLE
Screen to enter a code

### DISCUSSION THREAD

**Discussion by Mauricewillemstein** [Blackymas/NSPanel_HA_Blueprint#2179] [Ideas]
Hi, could it be possible to create a screen to enter a code, like 1234, to enable or disable an entety?
This would be great so kids cant enalbe or disable an entity on the ns-panel.
Something like an number keypad.

**Comment by edwardtfn** [comment]
What do you mean by enable or disable an entity?
Are you asking for a code based confirmation for actions from the panel? Any specific area? Could be a wake-up code and then everything will be available?

**Comment by Mauricewillemstein** [comment]
Hi, like an code to enable to use the entity.
Like a light or an lock.
Does not have to be fancy but you have to enter an code to use it.
Just like the confermation you have already have on the panel, just this time with an code

**Comment by rp2019433** [comment]
It seems to be a good feature. 
Thinking about a panel that is installed in a public area or restricted area, having a password to unlock the panel and use its entities is a good idea.
